CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ### Improvements
 * **core + dso-toolkit:** Header verbeteringen ([#1457](https://github.com/dso-toolkit/dso-toolkit/issues/1457))
+* **css + dso-toolkit + sources:** rich-content verhuizen naar @dso-toolkit/sources ([#1465](https://github.com/dso-toolkit/dso-toolkit/issues/1465))
```

Comment:
Zoek eens op "Verhui" in dit bestand, en kijk eens onder welke noemer die changes zijn gelogd

Reply:
Ah, die doen we onder `### Changed`, ik pas het aan